### DISCUSSION TITLE
listing on the new exchange and uniswap

### DISCUSSION THREAD

**Discussion by mrtaspinar** [oasisprotocol/community-forum#22] [Q&A]
@kostko Hi , 
Will Oasis network be listing new exchanges anytime soon ?
and
When can we buy with uniswap ?

Can you give me some information, please?
Have a good day

**Comment by tjanez** [comment]
> When can we buy with uniswap ?

I can elaborate on this part.

Second State has migrated Uniswap V1 to the Oasis-Eth ParaTime and it is up and running at https://uniswap.oasiseth.org/swap.

As was mentioned in [Oasis April Engineering Updates](https://medium.com/oasis-protocol-project/oasis-april-engineering-updates-19afa2bbf10e), the Foundation is funding 2 projects:

- A project migrating Uniswap V2 to the Oasis-Eth ParaTime. Code: https://github.com/dltswap. Demo: https://app-dltswap.herokuapp.com/.
- A project building DEX (name TBD) on the Oasis-Eth ParaTime.

In addition, there were multiple successful Oasis-Eth Hackathon projects building DEX and other DeFi-related projects:

- [Tulip Swap](https://tulipswap.io/) leveraging Uniswap V2 and SushiSwap.
- [RosebudSwap](https://rosebud.finance/#/swap) leveraging SushiSwap.
